indexer/reindex_bags_backlog.py

```diff
@@ -107,6 +107,22 @@ def get_latest_bags(dynamodb_client, table_name):
     return bags
 
 
+def get_bag(dynamodb_client, table_name, bag_id):
+    response = dynamodb_client.query(
+        TableName=table_name,
+        KeyConditionExpression="id = :bag_id",
+        ExpressionAttributeValues={
+            ":bag_id":{
+                "S":bag_id
+            }
+        }
+    )
+
+    items = response['Items']
+
+    return { bag_id:int(item['version']['N']) for item in items }
```

Comment:
This might be working by coincidence, but it relies on `items` being sorted in ascending order (and is a bit obfuscated, I think?). How about:

```python
max_version = max(int(item["version"]["N"] for item in response["Items"])
return {bag_id: max_version}
```

(Also I ditched the intermediate `items` variable; I get twitchy whenever I see variable names that are a single character apart.)